automl/cloud-client/delete_dataset.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
```

Comment:
This only makes sense if you plan on executing the script - there isn't a main in this script, so this should be removed.

Reply:
(same for the other scripts)